Source/WebCore/rendering/FloatingObjects.h

```diff
@@ -92,6 +92,10 @@ class FloatingObject {
     void clearOriginatingLine() { m_originatingLine = nullptr; }
     void setOriginatingLine(LegacyRootInlineBox& line) { m_originatingLine = line; }
 
+    bool containingBlockHasBlockEndMarginTrim() const;
+    void setHasTrimmedMargins(bool hasTrimmedMargins) { m_hasTrimmedMargins = hasTrimmedMargins; }
```

Comment:
This could be just a simple setHasTrimmedMargins() { m_hasTrimmedMargins = true; } since you don't go from trimmed to untrimmed.

Reply:
Changed